service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/NoticeOfProceedingsControllerAboutToSubmitTest.java

```diff
@@ -73,6 +81,54 @@ void shouldGenerateC6NoticeOfProceedingsDocument() throws Exception {
         assertThat(noticeOfProceedingBundle.getBinaryUrl()).isEqualTo(document.links.binary.href);
     }
 
+    @Test
+    void shouldMigrateJudgeAndLegalAdvisorWhenUsingAllocatedJudge() {
+        Document document = document();
+        DocmosisDocument docmosisDocument = DocmosisDocument.builder()
+            .bytes(PDF)
+            .documentTitle(C6_DOCUMENT_TITLE)
+            .build();
+
+        CallbackRequest callbackRequest = buildCallbackRequest();
+
+        Map<String, Object> templateData = createTemplatePlaceholders();
+
+        given(noticeOfProceedingsService.getNoticeOfProceedingTemplateData(any()))
+            .willReturn(templateData);
+        given(docmosisDocumentGeneratorService.generateDocmosisDocument(any(), any()))
+            .willReturn(docmosisDocument);
+        given(uploadDocumentService.uploadPDF(any(), any()))
+            .willReturn(document);
```

Comment:
This is used in method above too - could be extracted to a before block

Reply:
Not quite. In my latest test I do not need to test params so I use any(). While the above test it provides correct arguments.